Makefile

```diff
@@ -59,6 +59,7 @@ fail_release:
 trace_test:
 	env CI=true SKIP_FORMAT_CHECK=true FAIL_POINT=1 ${PROJECT_DIR}/ci-build/test.sh
 
+test: SHELL=/bin/bash -o pipefail
```

Comment:
Why? I don't see any pipes in this target.

Reply:
Good point. Will fix it.